packages/gestalt/src/TableSortableHeaderCell.js

```diff
@@ -38,33 +38,36 @@ export default function TableSortableHeaderCell(props: Props) {
 
   return (
     <TableHeaderCell colSpan={colSpan} rowSpan={rowSpan} scope={scope}>
-      <Touchable
-        onTouch={onSortChange}
-        onMouseEnter={() => setHovered(true)}
-        onMouseLeave={() => setHovered(false)}
-        onFocus={() => setFocused(true)}
-        onBlur={() => setFocused(false)}
-      >
-        <Box display="flex">
-          {children}
-          <Box
-            marginLeft={2}
-            dangerouslySetInlineStyle={{
-              __style: { visibility },
-            }}
-          >
-            <Icon
-              accessibilityLabel=""
-              icon={
-                status === 'active' && sortOrder === 'asc'
-                  ? 'sort-ascending'
-                  : 'sort-descending'
-              }
-              color={status === 'active' ? 'darkGray' : 'gray'}
-            />
+      <Box width="fit-content">
+        <Touchable
+          fullWidth={false}
+          onTouch={onSortChange}
+          onMouseEnter={() => setHovered(true)}
+          onMouseLeave={() => setHovered(false)}
+          onFocus={() => setFocused(true)}
+          onBlur={() => setFocused(false)}
+        >
+          <Box display="flex">
+            {children}
+            <Box
+              marginLeft={2}
```

Comment:
@PayalModi can we use `marginStart` instead of `marginLeft`? That way we support RTL out of the box:

marginLeft
![Screen Shot 2020-06-04 at 3 04 11 PM](https://user-images.githubusercontent.com/127199/83815549-45c2cc00-a675-11ea-859b-c55bd516c0d4.png)

marginStart (correct margin, even with RTL - Right to Left)

![Screen Shot 2020-06-04 at 3 04 03 PM](https://user-images.githubusercontent.com/127199/83815553-478c8f80-a675-11ea-89c4-51bc945db969.png)